src/services/posts.js

```diff
@@ -102,6 +104,16 @@ const assignTagToPost = async({postId , tagId}) => {
 	}
 	return postsTags
 }
+
+const fetchPost = async() => {
+	const result = await db.query('SELECT * FROM posts')
+	const feed = result.rows
+	if (!feed) {
```

Comment:
@beblicarl so we will not need to check this. even if there's no post, it will just be an empty array which is fine. no need for an error.

the condition will hardly execute as well because even if there's no feed it will be an `[]` and not `undefined` so this error will never throw anyway

Reply:
alright, i will work on that